client-projects/the-lumery/js/JobsComponent.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useEffect, useState } from "react";
+
+export const JobsContainer = () => {
+  const [isLoading, setLoading] = useState(false);
+  const [jobs, setJobs] = useState([]);
+
+  getJobsJSON(setLoading, setJobs);
+
+  return JobsDisplay(isLoading, jobs);
+};
+
+const getJobsJSON = async (setLoading, setJobs) => {
+  useEffect(async () => {
+    const response = await fetch(
+      "https://tlmw-external.azurewebsites.net/api/JobPostingsTrigger?code=mQJYB4t8rc6p6Zr-Khjbk5srngBr_BVdgdDWdzIsMgJdAzFuuX4RDw==&format=json",
+    );
+    if (response.ok) {
+      const json = await response.json();
+      setJobs(json.jobs);
+      setLoading(false);
+    }
+  }, []);
+};
+
+const JobsDisplay = (isLoading, jobJson) => {
+  return (
+    <div className="margin-small">
+      <div className="text-wrapper-hiring">
+        <h2 className="heading-style-h2">Today we're hiring</h2>
+        <div className="padding-bottom padding-small"></div>
+        <div className="text-body-medium">
+          Both data and Technology can be pretty flat without some bright minds
+          and big hearts to bring the magic. That's where you come in.
+        </div>
+      </div>
+      <div className="padding-bottom padding-large"></div>
+      {isLoading ? (
```

Comment:
Actually, can you make this the root of the component, and leave the heading and intro in Webflow for the client to update if they want?
> **Today, we’re hiring** 
> Both Data and Technology can be pretty flat without some bright minds and big hearts to bring the magic. That’s where you come in.